neuralop/layers/tests/test_local_fno_block.py

```diff
@@ -14,7 +14,7 @@ def test_LocalFNOBlock_resolution_scaling_factor():
     mlp_skip='linear'
     for dim in [1, 2, 3]:
```

Comment:
Instead of looping through here, let's add an 

`@pytest.mark.parametrize` call

Reply:
and you don't need call the test functions at the bottom, Pytest collects and runs them automatically

Reply:
Thanks @dhpitt! Yeah, sorry I added the test functions at the bottom just to be able to quickly test myself but forgot to remove them in my initial commit.